ledger/nomos-ledger/src/mantle/mod.rs

```diff
@@ -99,15 +99,15 @@ impl LedgerState {
         let mut balance = 0;
         for (op, proof) in tx.ops_with_proof() {
             match (op, proof) {
-                (Op::ChannelBlob(op), None) => {
+                (Op::ChannelBlob(op), Some(OpProof::Ed25519Sig(_sig))) => {
```

Comment:
Should we enforce that at type-level? So we would have some wrapper around and then we just accept the validated one here?